misalignedTest.h

```diff
@@ -0,0 +1,32 @@
+#ifndef MISALIGNEDTEST
+#define MISALIGNEDTEST
+
+#include <assert.h>
+
+void testColorPairValues(ColorPair &colorPair)
+{
+    assert(colorPair.number.at(0) == "1");
+    assert(colorPair.majorColorList.at(0) == "White");
+    assert(colorPair.minorColorList.at(0) == "Blue");
+    assert(colorPair.number.at(10) == "11");
+    assert(colorPair.majorColorList.at(10) == "Black");
+    assert(colorPair.minorColorList.at(10) == "Blue");
+    assert(colorPair.number.at(21) == "22");
+    assert(colorPair.majorColorList.at(21) == "Violet");
+    assert(colorPair.minorColorList.at(21) == "Orange");
+}
+
+void testColorMapAlignment(ColorPair &colorPairAlignment)
+{
+    int i = 0, j = 0;
+    for(i = 0; i < 25; i++)

```

Comment:
All we need to check is that each line has the separator in the same position. But we are iterating 25 * 25 times. Also the `25` is hard-coded. Ensure test code has same (high) quality and readability.